docs/layers.md

```diff
@@ -1354,55 +1451,688 @@ Depending on OpenStreetMap tagging, the following properties may be present for
+* `BR:PR` - **Paraná** region in Brazil
+* `BR:RJ` - **Rio de Janeiro** region in Brazil
+* `BR:RN` - **Rio Grande do Norte** region in Brazil
+* `BR:RO` - **Rondônia** region in Brazil
+* `BR:RR` - **Roraima** region in Brazil
+* `BR:RS` - **Rio Grande do Sul** region in Brazil state roads with prefix in (ERS, VRS, RSC)
+* `BR:SC` - **Santa Catarina** region in Brazil
+* `BR:SE` - **Sergipe** region in Brazil
+* `BR:SP:PLN` - **municipal roads in Paulínia** region in Brazil, with ref prefix in `PLN`
+* `BR:SP:SCA` - **municipal roads in São Carlos** region in Brazil with ref in `SCA`
+* `BR:SP` - **São Paulo** region in Brazil and access roads ref prefix of `SPA`
+* `BR:TO` - **Tocantins** region in Brazil
+* `BR:Trans-Amazonian` - **Trans-Amazonian** route in Brazil when network is BR and ref == `BR-230`
+* `BR` - Brazil fallback when operator in `Autopista Litoral Sul`, `Cart`, `DNIT`, `Ecovias`, `NovaDutra`, `Triângulo do Sol`, `Viapar`, `ViaRondon`
+* `CA:AB:primary` - **Alberta** region in Canada
+* `CA:AB:trunk` - **Alberta** region in Canada
+* `CA:AB` - **Alberta** region in Canada
+* `CA:BC:primary` - **British Columbia** region in Canada
+* `CA:BC:trunk` - **British Columbia** region in Canada
+* `CA:BC` - **British Columbia** region in Canada TODO: ???
+* `CA:MB:PTH` - **Manitoba** region in Canada
+* `CA:MB` - **Manitoba** region in Canada
+* `CA:NB` - **New Brunswich** region in Canada
+* `CA:NB2` - **New Brunswich** region in Canada network.startswith(`CA:NB`) and \ and refnum >= 100
+* `CA:NB3` - **New Brunswich** region in Canada network.startswith(`CA:NB`) and \ and refnum >= 200
+* `CA:NS:R` - **Nova Scotia** region in Canada 2x TODO: ???
+* `CA:NS:T` - **Nova Scotia** region in Canada 2x TODO: ???
+* `CA:NS` - **Nova Scotia** region in Canada 2x TODO: ???
+* `CA:NT` - **Northwest Territories** region in Canada
+* `CA:ON:primary` - **Ontario** region in Canada
+* `CA:ON` - **Ontario** region in Canada TODO: ???
+* `CA:PEI` - **Prince Edward Island** region in Canada
+* `CA:QC:A` - **Quebec** region in Canada
+* `CA:QC:R` - **Quebec** region in Canada
+* `CA:QC` - **Quebec** region in Canada TODO: ???
+* `CA:SK:primary` - **Sashsqatuan** region in Canada TODO: ???
+* `CA:SK` - **Sashsqatuan** region in Canada TODO: ???
+* `CA:transcanada` - **Canada Transcanada** highway and when ref)) when nat_name and nat_name.lower() == `trans-canada highway`
+* `CA:yellowhead` - **Canada Transcanda variant**
+* `CA:YT` - **Yukon Territory** region in Canada
+* `CD:RRIG` - Democratic Republic of the Congo if network == `CD:rrig`
+* `CH:motorway` - Switzerland
+* `CH:national` - Switzerland
+* `CH:regional` - Switzerland
+* `CL:national` - Chile
+* `CL:panamerican` - Chile
+* `CL:regional` - Chile
+* `CN:expressway:regional` - China (when expressway starts with S; normalized from `CN-expressways-regional`)
+* `CN:expressway` - China (when expressway starts with G; normalized from `CN-expressways)
```

Comment:
Missing backtick after `CN-expressways`?

Reply:
Fixed in ccc376f7bc3dda2d3be7fe0408206af0623f88e7.